### ISSUE TITLE
Are you aware of macOS native functionality to do thid?

### ISSUE THREAD

**Issue by dominickolbe** [Iomegan/Screenshot-Path-2#1]
Not sure if you know, but you can open the Screenshot app, go to Options, and change the location. You can even execute a script to make this change.

```bash
# Screenshot settings
defaults write com.apple.screencapture location -string "${HOME}/Downloads"
defaults write com.apple.screencapture type -string "png"
```

https://www.macworld.com/article/673251/how-to-change-where-screenshots-are-saved-on-a-mac.html

**Comment by Iomegan** [comment]
Yup, that's true. This app comes from a time with no UI for that, many years ago. 🙂

**Comment by dominickolbe** [comment]
No worries man, good work! Was just wondering about the "recent" commits. Currently building my dotfiles + automated macos setup script and was doing some research.
